editor/plugins/node_3d_editor_plugin.h

```diff
@@ -388,6 +399,28 @@ class Node3DEditorViewport : public Control {
 	void reset_fov();
 	void scale_cursor_distance(real_t scale);
 
+	struct _ShortcutCheckSet {
+		bool mod_pressed;
+		bool shortcut_not_empty;
+		int input_count;
+		ViewportNavMouseButton mouse_preference;
+		NavigationMode result_nav_mode;
+
+		_ShortcutCheckSet(){};
+
+		_ShortcutCheckSet(bool p_mod_pressed, bool p_shortcut_not_empty, int p_input_count, const ViewportNavMouseButton &p_mouse_preference, const NavigationMode &p_result_nav_mode) :
+				mod_pressed(p_mod_pressed), shortcut_not_empty(p_shortcut_not_empty), input_count(p_input_count), mouse_preference(p_mouse_preference), result_nav_mode(p_result_nav_mode) {
+		}
+	};
+
+	struct _ShortcutCheckSetComparator {
+		_FORCE_INLINE_ bool operator()(const _ShortcutCheckSet &A, const _ShortcutCheckSet &B) const {
+			return A.input_count > B.input_count;
+		}
+	};
```

Comment:
I don't recall seeing private structs starting with an underscore in Godot. We just name the like public structs as far as I know.

Reply:
Ah okay, I was following the `_RayResult` struct in the same file but you're right, it seems more common the other way elsewhere. I've renamed it now :) let me know if there's anything else I should adjust!